salesforce/bulkwrite.go

```diff
@@ -118,87 +123,20 @@ func (c *Connector) BulkWrite( //nolint:funlen,cyclop
 		return nil, fmt.Errorf("%w: %s", ErrUnsupportedMode, config.Mode)
 	}
 
-	// cretes batch upload job, returns json with id and other info
```

Comment:
The below code is moved to `BulkJob` so that we can use it in both bulk write and bulk delete with a few modifications.